patches/tModLoader/Terraria.IO/WorldFile.cs.patch

```diff
@@ -150,6 +150,23 @@
  				{
  					writer.Write(nPC2.active);
  					writer.Write(nPC2.netID);
+@@ -2181,14 +_,14 @@
+ 				int num3 = reader.ReadInt32();
+ 				Tile tile = Main.tile[num2, num3];
+ 				Sign sign;
+-				if (tile.active() && Main.tileSign[(int)tile.type])
++				if (true) //if (tile.active() && Main.tileSign[(int)tile.type]) 
```

Comment:
I'm not sure about this, it looks very volatile.

Reply:
I've approved it